manifests/port.pp

```diff
@@ -36,20 +36,17 @@
 #  }
 #
 define selinux::port (
-  $context  = undef,
+  $context,
+  $port,
   $protocol = undef,
-  $port     = undef,
-  $policy   = 'targeted'
 ) {
-  Exec {
-    path => '/bin:/sbin:/usr/bin:/usr/sbin',
-  }
 
-  if ( $context == undef ) or ( $port == undef ) {
```

Comment:
Yeah, I admit that the error messages here suck. I'm happy to merge this as-is, but I will more than likely change the behavior back in short order. So, up to you if you want to help shape it. :grinning: